pymc_marketing/model_config.py

```diff
@@ -392,9 +462,11 @@ def create_distribution(
     -------
     TensorVariable
         A PyMC random variable.
-
     """
-    dim_handler = create_dim_handler(kwargs.get("dims"))
+    if centered_hierarchy is False:
```

Comment:
if not centered_hierarchy:

Reply:
That generates problem when is `None` and we need to handle the case when is none in case don't exist in the dict because they are not using hierarchies. Related to next comments!